src/document.h

```diff
@@ -40,6 +40,8 @@ typedef struct {
   const char *name;  // Can either be char or RMString
   const char *path;
   RedisModuleString *text;
+  RedisModuleString **multiVal;
```

Comment:
Why not using Arr?

Reply:
I prefer C arrays for static arrays and use Arr for dynamic ones.